lighthouse-cli/performance-experiment/formatters/partials/config-panel.html

```diff
@@ -0,0 +1,286 @@
+  .config-panel {
+    margin-left: 0;
+    width: 100%;
+  }
+  .config-panel__header {
+    padding-right: 2px;
+  }
+}
+
+@media print {
+  .config-panel {
+    display: none;
+  }
+}
+
+#config-panel__tabs__tree-panel .request-blocking-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+  display: inline-block;
+  padding: 0 0 0 0px;
+  width: 18px;
+  height: 18px;
+  transform: translate(0px, 5px);
+  border: none;
+  flex: 0 0 auto;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 16px 16px;
+  background-color: transparent;
+}
+
+#config-panel__tabs__tree-panel .cnc-node.request__block .request-blocking-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" fill="#000000" height="24" viewBox="0 0 24 24" width="24"><path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"/><path xmlns="http://www.w3.org/2000/svg" d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+}
+
+#config-panel__tabs__tree-panel .cnc-node.request__block > .request-blocking-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" fill="#000000" height="24" viewBox="0 0 24 24" width="24"><path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"/><path xmlns="http://www.w3.org/2000/svg" d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z" fill="red" stroke="red"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+}
+
+.config-panel__tabs {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.config-panel__tabs__tab-bar {
+  height: 40px;
+  border-bottom: 1px solid #EBEBEB;
+  padding: 0 20px;
+}
+
+.config-panel__tabs__tab {
+  float: left;
+  display: inline-block;
+  color: black;
+  text-align: center;
+  text-decoration: none;
+  transition: 0.3s;
+  font-size: 17px;
+  line-height: 40px;
+  height: 40px;
+  padding: 0 10px;
+}
+
+.config-panel__tabs__tab:after {
+  content: '';
+  display: block;
+  margin: auto;
+  height: 3px;
+  width: 0px;
+  transform: translate(0px, -3px);
+  background: transparent;
+  transition: width .5s ease, background-color .5s ease;
+}
+
+.config-panel__tabs__tab.is-active:after {
+  width: 100%;
+  background: #7070fb;
+}
+
+.config-panel__tabs__tab:hover {
+  background-color: #ddd;
+}
+
+.config-panel__tabs__tab-panel {
+  display: none;
+  padding: 10px 20px;
+  overflow-y: scroll;
+  flex: 1 1 auto;
+}
+
+.config-panel__tabs__tab-panel.is-active {
+  display: block;
+}
+
+</style>
+
```

Comment:
Reuse critical request chain with some small changes.
The original critical request chain is almost unchanged.